src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerIssueAos.java

```diff
@@ -58,7 +63,13 @@ public SubmittedCallbackResponse submitted(final CaseDetails<CaseData, State> de
         log.info("Caseworker issue AOS submitted callback invoked");
 
         final CaseData caseData = details.getData();
-        noticeOfProceedingsNotification.send(caseData, details.getId());
+        final Long caseId = details.getId();
+
+        if (PERSONAL_SERVICE.equals(caseData.getApplication().getSolServiceMethod())) {
+            personalServiceNotification.send(caseData, caseId);
```

Comment:
it would be worth logging which notification we are sending for a particular case id

Reply:
I added logging to the PersonalServiceNotification component
https://github.com/hmcts/nfdiv-case-api/pull/418/commits/a3f51e6c5403cfeaf1c8e6576247c31a95c12248